packages/swapper/src/swappers/JupiterSwapper/utils/helpers.ts

```diff
@@ -0,0 +1,72 @@
+import { type ChainId, fromAssetId } from '@shapeshiftoss/caip'
+import type { Result } from '@sniptt/monads'
+import type { AxiosResponse } from 'axios'
+
+import type { SwapErrorRight } from '../../../types'
+import type { QuoteResponse } from '../models/QuoteResponse'
+import type { SwapInstructionsResponse } from '../models/SwapInstructionsResponse'
+import { jupiterSupportedChainIds } from './constants'
+import { jupiterService } from './jupiterService'
+
+export const isSupportedChainId = (chainId: ChainId): chainId is ChainId => {
+  return jupiterSupportedChainIds.includes(chainId)
+}
+
+const JUPITER_TRANSACTION_MAX_ACCOUNTS = 54
+
+type GetJupiterQuoteArgs = {
+  apiUrl: string
+  sourceAsset: string
+  destinationAsset: string
+  commissionBps: string
+  amount: string
+  slippageBps?: string
+}
+
+type GetJupiterSwapArgs = {
+  apiUrl: string
+  fromAddress: string
+  rawQuote: unknown
+  toAddress?: string
+  useSharedAccounts: boolean
+  wrapAndUnwrapSol?: boolean
```

Comment:
suggestion: remove me, we never pass it and always leverage `wrapAndUnwrapSol = true`, so can just pass:
`wrapAndUnwrapSol: true` directly in the `jupiterService.post()` call

Reply:
Not removed but consumed for the `wrappedSol` usecase in [4e76e81](https://github.com/shapeshift/web/pull/8120/commits/4e76e8115223d91bd602f7d9ceed80c94c77ea15)